src/patternfly/components/Hint/examples/Hint.md

```diff
@@ -0,0 +1,153 @@
+---
+title: 'Hint'
+section: components
+beta: true
+cssPrefix: pf-c-hint
+---
+
+## Examples
+```hbs title=Default-with-header
+{{#> hint}}
+  {{#> hint-header}}
+    Do more with Find it Fix it capabilities
+  {{/hint-header}}
+  {{#> hint-body}}
+    Upgrade to Red Hat Smart Management to remediate all your systems across regions and geographies.
+  {{/hint-body}}
+  {{#> hint-actions}}
```

Comment:
why do the actions come last?

Reply:
Moved them to after the header (similar to what is in the card)

Reply:
This seems fine to me, the modal and popover are the same way (action comes first), though I'm not sure if that's specific to a close button in an element like that or if it can apply to general actions. @jessiehuff wdyt?